lib/fs_utils/file_list.js

```diff
@@ -121,11 +123,17 @@ class FileList extends EventEmitter {
     return this.assets.find(file => file.path === path);
   }
 
-  compileDependencyParents(path) {
+  findStatic(path) {
+    path = normalizePath(path);
+    return this.staticFiles.get(path);
+  }
+
+  compileDependencyParents(path, isStatic) {
     path = normalizePath(path);
     const paths = [];
     const parents = [];
-    this.files.forEach((dependent, fpath) => {
+    const files = isStatic ? this.staticFiles : this.files;
```

Comment:
The dependency tracking logic is similar for both, this check as to where pick the files from is about the only thing that differs in this regard